server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/deltas.ts

```diff
@@ -147,6 +147,7 @@ export function create(
 			const from = stringToSequenceNumber(request.query.from);
 			const to = stringToSequenceNumber(request.query.to);
 			const tenantId = getParam(request.params, "tenantId") || appTenants[0].id;
+			const caller = request.query.token?.toString().length > 0 ? "external" : "scribe";
```

Comment:
There are two issues:
1. I think the logic here is reversed. As of now, the calls with `token` is actually internal calls from scribe. 
2. I think instead of we assign logic internally, we can leverage this logic to the callers maybe? It will be more flexible, for example. you can add &caller=scribe from scribe, or &caller=oss, or even &caller=deli (If one day we need). What you think?

Reply:
Thats a good idea to send this as a query param, will update